homeassistant/components/august/camera.py

```diff
@@ -91,24 +89,9 @@ async def async_camera_image(
         self._update_from_data()
 
         if self._image_url is not self._detail.image_url:
-            self._image_url = self._detail.image_url
-            self._content_token = self._detail.content_token or self._content_token
-            _LOGGER.debug(
-                "calling doorbell async_get_doorbell_image, %s",
-                self._detail.device_name,
+            self._image_content = await self._data.async_get_doorbell_image(
+                self._device_id, self._session, timeout=self._timeout
             )
-            try:
-                self._image_content = await self._detail.async_get_doorbell_image(
-                    self._session, timeout=self._timeout
-                )
-            except ContentTokenExpired:
-                if self._data.brand == Brand.YALE_HOME:
-                    _LOGGER.debug(
-                        "Error fetching camera image, updating content-token from api to retry"
-                    )
-                    await self._async_update()
-                    self._image_content = await self._detail.async_get_doorbell_image(
-                        self._session, timeout=self._timeout
-                    )
+            self._image_url = self._detail.image_url
```

Comment:
now set after since the expired token may cause a refresh and the url could change